modal_proto/api.proto

```diff
@@ -36,7 +36,7 @@ enum ClientType {
   CLIENT_TYPE_WORKER = 2; // deprecated
   CLIENT_TYPE_CONTAINER = 3;
   CLIENT_TYPE_SERVER = 4; // deprecated
-  CLIENT_TYPE_WEB_SERVER = 5;
+  CLIENT_TYPE_WEB_SERVER = 5; // deprecated
```

Comment:
i think we can maybe use `[deprecated=true]` for these

Reply:
Nice I didn't realize that works!